SIDFactoryII/source/runtime/editor/editor_facility.cpp

```diff
@@ -100,9 +100,10 @@ namespace Editor
 		SIDConfiguration sid_configuration; // Default settings are applicable
 
 		const bool sid_use_resample = GetSingleConfigurationValue<ConfigValueInt>(config, "Sound.Emulation.Resample", 1) != 0;
+		const int sid_sample_frequency = GetSingleConfigurationValue<ConfigValueInt>(config, "Sound.Emulation.SampleFrequency", 44100);
 		sid_configuration.m_eSampleMethod = sid_use_resample ? SID_SAMPLE_METHOD_RESAMPLE_INTERPOLATE : SID_SAMPLE_METHOD_INTERPOLATE;
 		sid_configuration.m_eModel = SID_MODEL_6581;
-
+		sid_configuration.m_nSampleFrequency = sid_sample_frequency;
```

Comment:
Could you add a logging statement?

```c++
Logging::instance().Info("Sound.Emulation.SampleFrequency set to %d", sid_sample_frequency);
```

This way we can see what actual value is being set for the frequency.

Reply:
Done